packages/down-up-mixin/down-up-mixin.ts

```diff
@@ -0,0 +1,77 @@
+import { PropertyValues } from 'lit-element';
+import { Constructor, applyMixin, wasApplied } from '@vaadin/mixin-utils';
+import { DownUpClass } from './down-up-class';
+
+export const DownUpMixin = <T extends Constructor<DownUpClass>>(
+  base: T
+): T & Constructor<DownUpClass> => {
+  if (wasApplied(DownUpMixin, base)) {
+    return base;
+  }
+
+  class DownUp extends base {
+    protected firstUpdated(props: PropertyValues) {
+      super.firstUpdated(props);
+
+      if (this._downUpTarget instanceof HTMLElement) {
+        this._downUpTarget.addEventListener('mousedown', (event: MouseEvent) => {
+          this._onMouseDown(event);
+        });
+
+        this._downUpTarget.addEventListener('touchstart', (event: TouchEvent) => {
+          this._onTouchStart(event);
+        });
+
+        this._downUpTarget.addEventListener('touchend', (event: TouchEvent) => {
+          this._onTouchEnd(event);
+        });
+      }
+    }
+
+    protected get _downUpTarget() {
+      return this;
+    }
```

Comment:
Here's my suggestion on how to make it simpler to override `_downUpTarget` with an element in the component template. Also see my related comment in the tests where this approach is used in my suggestion.

```suggestion
  class DownUp extends base {
    protected _downUpTarget?: HTMLElement;

    protected firstUpdated(props: PropertyValues) {
      super.firstUpdated(props);

      this._initDownUpTarget();

      if (this._downUpTarget instanceof HTMLElement) {
        this._downUpTarget.addEventListener('mousedown', (event: MouseEvent) => {
          this._onMouseDown(event);
        });

        this._downUpTarget.addEventListener('touchstart', (event: TouchEvent) => {
          this._onTouchStart(event);
        });

        this._downUpTarget.addEventListener('touchend', (event: TouchEvent) => {
          this._onTouchEnd(event);
        });
      }
    }

    protected _initDownUpTarget() {
      if (this._downUpTarget === undefined) {
        this._downUpTarget = this;
      }
    }
```

Reply:
Updated the tests but kept the getter as it is.